tests/models/bloom/test_tokenizer_bloom.py

```diff
@@ -0,0 +1,97 @@
+import unittest
```

Comment:
Should be merged with the other tokenization test file.

Reply:
Fixed in e580cb5a5e848b27c40eb835bc93159b0dbfe182